client/packages/host/src/components/Navigation/ReplenishmentNav.tsx

```diff
@@ -42,6 +45,7 @@ export const ReplenishmentNav: FC = () => {
               .addPart(AppRoute.InternalOrder)
               .build()}
             text={t('internal-order')}
+            badgeProps={getBadgeProps(requestCount?.draftCount)}
```

Comment:
The badge looks great. Find it a bit weird to have it just for `Internal Orders` though... 
https://github.com/openmsupply/open-msupply/issues/1647#issuecomment-1529296336